.gitignore

```diff
@@ -67,4 +67,7 @@ doc/lightning*.[1578]
 
 # Rust targets
 target
-Cargo.lock
\ No newline at end of file
+Cargo.lock
+
+# other binary
+fp16
```

Comment:
`devtools/.gitignore` would be a better place for this

Reply:
Good catch thanks!